src/ai/backend/manager/models/user.py

```diff
@@ -658,7 +660,19 @@ async def _post_func(conn: SAConnection, result: Result) -> Row:
                 groups.c.type == ProjectType.MODEL_STORE
             )
             model_store_gid = (await conn.execute(model_store_query)).first()["id"]
-            gids_to_join = [*group_ids, model_store_gid]
+
+            async def get_group_ids(conn, g_names):
+                g_ids = await asyncio.gather(*[
+                    conn.scalar(sa.select([groups.c.id]).where(groups.c.name == name))
+                    for name in g_names
+                ])
+                for name, g_id in zip(g_names, g_ids):
+                    if g_id is None:
+                        raise GroupNotFound(f"Group '{name}' does not exist")
+                return g_ids
+
+            default_group_ids = await get_group_ids(conn, props.default_groups)
+            gids_to_join = [*default_group_ids, *group_ids, model_store_gid]
```

Comment:
We don't need to add new `default_groups` parameter in the API.
Reuse the `group_ids` parameter as intended.